charts/internal/shoot-storageclasses/templates/storageclasses.yaml

```diff
@@ -18,4 +18,13 @@ metadata:
 allowVolumeExpansion: true
 provisioner: ebs.csi.aws.com
 volumeBindingMode: WaitForFirstConsumer
+---
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  name: default
```

Comment:
What about?

```suggestion
  name: gardener-default
```

Reply:
done

Reply:
Why `gardener-default`? Is `default` somehow reserved? Can't we keep it similar to the `default` StorageClass?

Reply:
I'm fine with both. Many examples/guides call their storage classes "default" which conflicts with Gardener's default storage class. But there's no "hard" reason and just this soft criteria that came to my mind. If you prefer `default` over `gardener-default` then go ahead, I'm fine.

Reply:
Ok, then I would indeed go with `default` as the name for the `VolumeSnapshotClass` to make it consistent with the `StorageClass` and also other examples :)